modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase

```diff
@@ -49,25 +47,67 @@ definition {
 	@description = "LPS-164617. Assert that it's possible to make changes in the FDS "
 	@priority = "4"
 	test CanBeEditted {
-
-		// To-do CanBeEditted Autom. test
-
 	}
 
 	@description = "LPS-164616. Assert that it's possible to create A new custom view and selected as current view"
 	@priority = "4"
 	test CanBeSaved {
-
-		// To-do CanBeSaved Autom. test
-
 	}
 
 	@description = "LPS-164618. Assert that it's possible can edit a new instance was previously created"
 	@priority = "4"
 	test CanCreateMultiple {
+		task ("And Given A custom view is created and selected") {
+			FDSCustomView.createNewView(key_nameView = "FrontendDataSet View");
+
+			FDSCustomView.assertCustomView(key_nameCustomView = "FrontendDataSet View");
+		}
+
+		task ("When Make some changes to dataset display (filter, sorting, page size or visible columns if data display in table)") {
+			FDSFilters.editFilters(
+				key_disableFilters = "Blue,Green,Yellow",
+				key_enableFilters = "Red",
+				key_filter = "Color");
+
+			Click(
+				key_text = "order-arrow-down",
+				locator1 = "Icon#ANY");
+
+			Click(
+				key_text = "order-arrow-down",
+				locator1 = "Icon#ANY");
 
-		// To-do CanBeSaved Autom. test
+			FDSCustomView.selectColumn(key_optionName = "Description");
 
+			Pagination.changePagination(itemsPerPage = "4 items");
+		}
+
+		task ("And Click Save as button, provide a name") {
+			FDSCustomView.createNewView(key_nameView = "FrontendDataSet View 2");
+		}
+
+		task ("Then A new custom view is created, and selected as current view") {
+			FDSCustomView.assertCustomView(key_nameCustomView = "FrontendDataSet View 2");
+
+			FDSCustomView.changeCustomView(key_optionName = "FrontendDataSet View");
+
+			FDSCustomView.assertFilterSummaryLabel(key_filter = "Color: Blue, Green, Yellow");
+
+			FDSCustomView.viewColumn(key_field = "Description");
+
+			FDSCustomView.changeCustomView(key_optionName = "FrontendDataSet View 2");
+
+			Pagination.viewResults(results = "Showing 1 to 4 of 25");
+
+			FDSCustomView.assertFilterSummaryLabel(key_filter = "Color: Red");
+
+			FDSCustomView.viewNoColumn(key_field = "Description");
+
+			FDSCustomView.compareValue(
+				key_column = "2",
+				key_index1 = "1",
+				key_index2 = "2");
```

Comment:
we can refactor this next time to make it more readable if the test fails here.

Reply:
I tried to use the names of the dataset's columns in the compareValue macro, but it didn't work because the dataset is organized by rows and I didn't know how to connect the table cell value with its column name. 
for example, in the html code, the ID header doesn't have any parent relation with the sample's id cells. 
so I just improved the macro variables to turn it more legible to who would read it, but it still only works using the index of the columns.